readthedocs-theme/templates/readthedocs/homepage.html

```diff
@@ -12,24 +12,24 @@
 
         <div class="row">
 
-          <div class="column seven wide computer nine wide tablet sixteen wide mobile">
-            <h1 class="ui header massive">
+          <div class="column seven wide computer twelve wide tablet sixteen wide mobile">
+            <h1 class="ui header massive center align text tablet mobile">
               <span class="ui header">Push your docs, <br>we'll do the rest</span>
             </h1>
-            <div class="ui basic vertical padded huge segment">
+            <div class="ui basic vertical padded huge segment center align text tablet mobile">
               <p>
                 Make documentation part of your project workflow with simplified processes that automate building, versioning and hosting of your docs.
               </p>
             </div>
             
-            <div>
+            <div class="center align text tablet mobile">
               <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Get started!</a>
               <a href="./product.html" class="ui button large secondary basic">Learn more</a>
             </div>
           
           </div>
 
-          <div class="column seven wide computer nine wide tablet sixteen wide mobile">
+          <div class="seven wide computer only column">
```

Comment:
I assume you meant this instead:

```suggestion
          <div class="seven wide computer only computer">
```

Reply:
No. The classes `computer only` are order sensitive and need to be in that order. Also, what you suggest repeats the word computer and removes column.

Reply:
Yeah I thought the order was `only computer`.